clang/tools/include-mapping/cppreference_parser.py

```diff
@@ -232,4 +247,4 @@ def GetSymbols(parse_pages):
     finally:
         pool.terminate()
         pool.join()
-    return sorted(symbols)
+    return sorted(set(symbols))
```

Comment:
each symbol should already have a unique name, why do we need this conversion to a `set` here?